tools/templates/tools/usertool_detail.jinja

```diff
@@ -92,17 +88,24 @@
         {% endfor %}
         </div>
         <h3>{{ _("Cleared") }}</h3>
-        {% set cleared_count = tool.permissions.count() %}
-        {% set cleared_display_count = 6 %}
-        {% for item in tool.permissions.all()[:cleared_display_count] %}
-            <div class="list-group-item">
-                {{ object_link(item.cleared_user, "profile-link") }}<br>
-                <small class="font-weight-light text-muted">{{ item.created|date("SHORT_DATETIME_FORMAT") }}</small>
-            </div>
-            {% if loop.last and cleared_count > cleared_display_count %}
-                <a class="list-group-item list-group-item-action list-group-item-primary" href={{ url('tools:cleared', pk=tool.pk) }}>View all cleared People</a>
-            {% endif %}
-        {% endfor %}
+        {% if tool.Clearance(tool.clearance) == tool.Clearance.none %}
+            No Clearence Needed
+        {% else %}
+            {% set cleared_count = tool.permissions.count() %}
+            {% set cleared_display_count = 6 %}
+            {% for item in tool.permissions.all()[:cleared_display_count] %}
+                <div class="list-group-item mb-4">
```

Comment:
Could you remove the mb-4 and wrap the `.list-group-items` in a `.list-group`? It looks like I forgot to do that when originally making it.